ksql-rest-app/pom.xml

```diff
@@ -97,6 +97,50 @@
 
     <build>
         <plugins>
+
+            <!-- Grab the web interface -->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>wagon-maven-plugin</artifactId>
+                <version>1.0</version>
+                <executions>
+                    <execution>
+                        <id>download-ksql-experimental-ui</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>download-single</goal>
+                        </goals>
+                        <configuration>
+                            <url>https://github.com/confluentinc</url>
```

Comment:
any reason why this is not just picked up from maven as a dependency? I'm not sure it would be any easier, just wondering?

Reply:
not sure how I would do that.....

Reply:
@dguy : The experimental UI doesn't publish to maven central (or another mvn repo) right now, which I suppose is the reason @bluemonk3y uses this workaround.

Reply:
yep - we are short circuiting the packaging process to get it bundled as part of the ksql-rest-app.jar